tests/system/test_cluster/test_agent_files_deletion/test_agent_files_deletion.py

```diff
@@ -56,21 +81,22 @@ def test_agent_files_deletion(register_agent):
             assert result, f'This file should exist in {host} but could not be found: ' \
                            f'{file["path"].format(id=agent_id, name=agent_name)}'
 
-    # Check that agent information exists in global.db
+    # Check that agent information is in the wdb socket
     for host in managers_hosts:
-        for query in db_queries:
-            result = host_manager.run_command(
-                host,
-                f'sqlite3 {join(WAZUH_PATH, "queue", "db", "global.db")} "{query.format(id=agent_id, name=agent_name)}"'
-            )
-            assert result, f'This db query should have returned something in {host}, but it did not: ' \
-                           f'{query.format(id=agent_id, name=agent_name)}'
+        for query in queries:
+            result = host_manager.run_command(host,
+                                              f"{WAZUH_PATH}/framework/python/bin/python3.9 "
+                                              f"{script_path}/get_wdb_agent.py "
+                                              f"'{query.format(id=agent_id)}'")
+        assert result, f'This db query should have returned something in {host}, but it did not: {result}'
```

Comment:
```suggestion
        for query in queries:
            result = host_manager.run_command(host,
                                              f"{WAZUH_PATH}/framework/python/bin/python3.9 "
                                              f"{data_path}/get_wdb_agent.py "
                                              f"'{query.format(id=agent_id)}'")
        assert result, f"This db query should have returned something in {host}, but it did not: {result}"
```

We are checking just the result of the last query